data/json/scenarios/scenarios.json

```diff
@@ -197,5 +197,16 @@
     "start_name": "Bottom of a mine",
     "professions" : ["miner"],
     "allowed_locs" : ["mine_finale"]
+  },
+  {
+    "type": "scenario",
+    "ident": "fema_help",
+    "name": "Challenge-FEMA Death Camp",
+    "points" : -8,
```

Comment:
Not worth anywhere near 8 points as "death camps" are currently filled with unevolved zeds and have incredibly convenient fences you can just go over to lose the chasers. It's much safer than burning building and infected. It's worth 4 points at most.